public/css/me.css

```diff
@@ -52,49 +65,98 @@ button:hover {
 
 .blurb {
 	margin-bottom: 20px;
-}
+} */
 
 /* APPLICATION STATUS */
 
-.background {
+/* .background {
 	position: relative;
 	width: 510px;
 	margin: 40px auto;
 	padding: 20px 70px;
 	border-radius: 10px;
-}
+} */
```

Comment:
Again, you can just delete code if you don't need it for this year. Cleaner code is always better from a maintenance perspective